modules/network/ip/nibble.go

```diff
@@ -127,6 +155,22 @@ func (n *Nibble) WGLL() *net.IPNet {
 	}
 }
 
+// WGIP4
+func (n *Nibble) WGIP4() *net.IPNet {
```

Comment:
actually this function shouldn't be called anywhere -> drop?
(as it is the same code as EPPubIP4R)